pkg/plugin/query.go

```diff
@@ -150,6 +154,14 @@ func (q *Query) addMetadataToMultiFrame(frame *data.Frame) {
 		frame.Fields[1].Config = &data.FieldConfig{DisplayNameFromDS: customName}
 	}
 
+	if frame.Meta == nil {
+		frame.Meta = &data.FrameMeta{}
+	}
+
+	frame.Meta.Custom = &CustomMeta{
```

Comment:
I don't understand how it is used after. Could you please elaborate on this a bit? 
Could it collide with trace metadata at https://github.com/VictoriaMetrics/victoriametrics-datasource/blob/c5c960ab6b5c4b699dad441c77d0e115f8625554/pkg/plugin/response.go#L149-L155 ?